packages/client/src/containers/ProductView/ProductView.container.js

```diff
@@ -26,7 +29,54 @@ export const ProductView = () => {
   const navigateBack = () => {
     navigate(-1);
   };
+  const [orderValue, setOrderValue] = useState({
+    size: '',
+    color: '',
+    quantity: '',
+  });
+  const showSelectedValue = (variantType, variantValue) => {
+    if (variantType === 'color') {
+      setOrderValue((preValue) => {
+        return { ...preValue, color: variantValue };
+      });
+    }
+    if (variantType === 'size') {
+      setOrderValue((preValue) => {
+        return { ...preValue, size: variantValue };
+      });
+    }
+    if (variantType === 'quantity') {
+      setOrderValue((preValue) => {
+        return { ...preValue, quantity: variantValue };
+      });
+    }
```

Comment:
You can achieve the same thing with `setOrderValue((prev) => ({ ...prev, [variantType]: variantValue }))`.

Reply:
I have changed.